types/types.go

```diff
@@ -999,6 +1028,7 @@ func CreateDefaultDisk(dataPath string, storageReservedPercentage int64) (map[st
 			DefaultDiskPrefix + util.RandomID(): {
 				Type:              longhorn.DiskTypeBlock,
 				Path:              dataPath,
+				DiskDriver:        longhorn.DiskDriverNone,
```

Comment:
Sorry? Shouldn't this be `longhorn.DiskDriverAuto`?

Reply:
Filesystem-type disk doesn't have different drivers, so `DiskDriverNone` is fine here.

Reply:
I missed this part. Thank you.